governance/default.go

```diff
@@ -562,7 +564,10 @@ func (g *Governance) initializeCache() error {
 	if err != nil {
 		return ErrNotInitialized
 	}
+	g.idxCacheLock.Lock()
 	g.idxCache = indices
+	g.idxCacheLock.Unlock()
+
 	// Put governance items into the itemCache
 	for _, v := range indices {
```

Comment:
although it is same, would it be better use g.idxCache instead of indices?

Reply:
@laggu Good point. Thanks!

Reply:
Isn't locking needed here? (at least RLock())

Reply:
@ehnuje initializeCache is called only once when a node starts while calling `NewGovernance()`. So I don't think a locking is needed for `indices`.

Reply:
@ehnuje I removed lock on `g.idxCacheLock` in `initializeCache()` because it is redundant as we discussed. Please take a look.